### ISSUE TITLE
Zoomview and segments click events getting fired simultaneously

### ISSUE THREAD

**Issue by NinjaCoderDotCom** [bbc/peaks.js#490]
Hi @chrisn 

In my project, I need both `segments.click` and `zoomview.click`. When I click on segments, `zoomview.click` is also getting fired. I think only `segments.click` should fire in this case. I have tried in both of the cases: `enableSeeking=true` & `enableSeeking=false`  Do correct me if I am wrong.

**Comment by chrisn** [comment]
Yes, you'll get both events. The `enableSeek()` functions don't affect this. I'll need to think about how to handle this..

**Comment by chrisn** [comment]
I have added a `preventViewEvent()` method, so you can do this:

```javascript
peaks.on('segments.click', function(event) {
  event.preventViewEvent(); // Stops the zoomview.click event from being fired.
});
```

**Comment by NinjaCoderDotCom** [comment]
Hi, @chrisn 
Thanks. Have you released the new version along with these changes?

**Comment by chrisn** [comment]
I have just released v3.0.0-beta.11. (I hope to get v3.0.0 out at some point soon.)

**Comment by NinjaCoderDotCom** [comment]
Thanks, @chrisn for the support.
